impeller/display_list/dl_dispatcher.cc

```diff
@@ -1104,9 +1104,23 @@ void DlDispatcher::drawDisplayList(
 void DlDispatcher::drawTextBlob(const sk_sp<SkTextBlob> blob,
```

Comment:
I don't think it's strictly necessary, but adding a playground test in `display_list_unittests.cc` would be nice as a jumping off point for debugging in the future.

Reply:
Good idea, will do.

Reply:
Done